SwiftStyleGuide.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SwiftStyleGuide.swift
+//  PurchasesCoreSwift
+//
+//  Created by Andrés Boedo on 7/12/21.
```

Comment:
We probably should add a header:
```
/*
 *  Copyright RevenueCat Inc. All Rights Reserved.
 *
 *  Licensed under the MIT License (the "License");
 *  you may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at
 *
 *      https://opensource.org/licenses/MIT
 */
 ```

Reply:
do you mean specifically for this file, or as a template for all files?

Reply:
I think as a template for all files, and we should just update them all as we go along.

Reply:
🧡